.changelog/2444.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cli/user-modify: Return help on malformed command
```

Comment:
I think our prefix is usually just `cli:` and specify the "user modify" in the text